### DISCUSSION TITLE
Add Clip skip and eta_noise_seed_delta to the UI

### DISCUSSION THREAD

**Discussion by srgank67** [anapnoe/stable-diffusion-webui-ux#61] [Q&A] [answered]
Hi.
I'm switching my web-ui to this fork that looks incredible but I'm not able to put in the main UI the sliders foe the CLIP skip and eta noise seed delta. Anyone knows how to do it?
![image](https://user-images.githubusercontent.com/106804067/232710917-fcc227fc-fbe6-414f-95cb-e33e895c82c0.png)

**Comment by anapnoe** [comment, marked as answer]
you need to click the menu button with the + sign on the right to add it to the quickmenu
then from the top navigation menu click the 3 dots top right to open the quicksettings
you can add as many as you like 
you can reorder with drag n drop once you open the quicksettings click on the top left icon to enable reorder

**Reply by srgank67** [reply]
Ooh! I did not reallice. Many thanks!

**Reply by anapnoe** [reply]
can you mark this as answered you are welcome

**Reply by N2O1138** [reply]
I had clicked that on a couple settings and had no idea what it did besides make them disappear, thanks for clearing that up!
